gradle-plugin/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.4.1-bin.zip
```

Comment:
`6.5` is [out](https://gradle.org/releases/#close-notification) already. Any reason to stay on `6.4.1`?

Reply:
> `6.5` is [out](https://gradle.org/releases/#close-notification) already. Any reason to stay on `6.4.1`?

got some issues with 6.5 before, just wanted to proceed with 6.4.1 that we're actively using at work now.

Reply:
Ok, let's play it safe until it's ready in 6.4.1 and we can try the bump later.